sakila_project.sql

```diff
@@ -0,0 +1,70 @@
+# 1 Get 10 cities in descending alphabetical order.
+Select city
+From sakila.city
+order by city DESC
+Limit 10;
+
+#2. Get all films with "airplane" in the title.
+Select * From sakila.film
+Where title Like '%airplane%';
+
+#3. Get the highest payment amount.
+Select max(amount)
+From sakila.payment;
+
+#4. Get the number of records in the customer table for store id #1.
+Select * From sakila.customer 
+Where store_id = 1;
+
+#5. Get all payment records for customer with email address "NANCY.THOMAS@sakilacustomer.org"
+Select * From sakila.customer
+wHERE email Like "NANCY.THOMAS@sakilacustomer.org";
+
+#6. Use a View to get the film info for actor Bob Fawcett.
+
+Select * From sakila.actor_info
+Where first_name Like 'Bob' And last_name Like 'Fawcett';
+
+#7. Use a Stored Procedure to get the 4 inventory ids for the film "Alien Center" at Store #2. 
+Select * From sakila.film
```

Comment:
This should select into a variable. Select only the ID column (* selects all columns)